CoreWiki/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -21,14 +21,20 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
             modelBuilder.Entity("CoreWiki.Models.Article", b =>
                 {
-                    b.Property<string>("Topic")
+                    b.Property<int>("Id")
                         .ValueGeneratedOnAdd();
 
                     b.Property<string>("Content");
 
-                    b.Property<DateTime>("Published");
+                    b.Property<DateTime>("PublishedDateTime")
+                        .HasColumnName("Published");
+
+                    b.Property<string>("Slug");
+
+                    b.Property<string>("Topic")
```

Comment:
Topic and Slug should be indexed to allow for faster searching.  Also, we should mark these fields as unique

Reply:
Great idea